queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -4,7 +4,7 @@
 # Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
 # SPDX - License - Identifier: GPL-3.0-or-later
 # ############################################################################### #
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
This shebang won't work with the space there

Reply:
Yes, this is more of a fat finger introduced problem from me XD Thanks for noticing it! 👍